generated-source-files.mk

```diff
@@ -1,6 +1,6 @@
 m3x_package          := github.com/m3db/m3x
 m3x_package_path     := $(gopath_prefix)/$(m3x_package)
-m3x_package_min_ver  := 29bc232d9ad2e6c4a1804ccce095bb730fb1c6bc
+m3x_package_min_ver  := 4018c53653ab98801d09d2b88b5e0bb9e3be6dca
```

Comment:
reminder, this needs to be updated once https://github.com/m3db/m3x/pull/141 lands

Reply:
done